### PR TITLE
dev/reference, v3.0/reference, v2.1/reference: update tidb lightning download link

### PR DESCRIPTION
**Pull request by ericsyh**
### What is changed, added or deleted? 

The download link for TiDB Lightning is out of date. Submit this PR to update the download link for TiDB Lightning.

### Related PR

https://github.com/pingcap/docs/pull/1337

### Which version does your change affect? 

Apply in dev, 3.0 and 2.1.

### Checklist 

- [ ] Add a new file to `TOC.md`
- [x] No Tab spaces anywhere 
- [ ] Leave a blank line both before and after a code block
- [ ] Keep the first level heading consistent with `title` in metadata
- [ ] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

File: v2.1/reference/tools/download.md [hunk 1]
```diff
@@ -7,16 +7,24 @@ category: reference
...
 | 安装包 | 操作系统 | 架构 | SHA256 校验和 |
 |:---|:---|:---|:---|
-| [tidb-latest-linux-amd64.tar.gz](http://download.pingcap.org/tidb-latest-linux-amd64.tar.gz) (TiDB-Binlog, Lightning) | Linux | amd64 |[tidb-latest-linux-amd64.sha256](http://download.pingcap.org/tidb-latest-linux-amd64.sha256)|
-| [tidb-binlog-kafka-linux-amd64.tar.gz](http://download.pingcap.org/tidb-binlog-kafka-linux-amd64.tar.gz)（Kafka 版本的 TiDB-Binlog）| Linux | amd64 |[tidb-binlog-kafka-linux-amd64.sha256](http://download.pingcap.org/tidb-binlog-kafka-linux-amd64.sha256)|
+| [tidb-latest-linux-amd64.tar.gz](http://download.pingcap.org/tidb-latest-linux-amd64.tar.gz) (TiDB Binlog) | Linux | amd64 |[tidb-latest-linux-amd64.sha256](http://download.pingcap.org/tidb-latest-linux-amd64.sha256)|
+| [tidb-binlog-kafka-linux-amd64.tar.gz](http://download.pingcap.org/tidb-binlog-kafka-linux-amd64.tar.gz)（Kafka 版本的 TiDB Binlog）| Linux | amd64 |[tidb-binlog-kafka-linux-amd64.sha256](http://download.pingcap.org/tidb-binlog-kafka-linux-amd64.sha256)|
+
+## TiDB Lightning
+
+如需下载最新版本的 [TiDB Lightning](/reference/tools/tidb-lightning/overview.md), 点击下表中的下载链接即可。
```
Comment (lilin90) [inline v2.1/reference/tools/download.md]:
```suggestion
如需下载最新版本的 [TiDB Lightning](/reference/tools/tidb-lightning/overview.md)，点击下表中的下载链接即可。
```

File: v3.0/reference/tools/download.md [hunk 2]
```diff
@@ -8,16 +8,24 @@ aliases: ['/docs-cn/tools/download/']
...
 | 安装包 | 操作系统 | 架构 | SHA256 校验和 |
 |:---|:---|:---|:---|
-| [tidb-latest-linux-amd64.tar.gz](http://download.pingcap.org/tidb-latest-linux-amd64.tar.gz) (TiDB-Binlog, Lightning) | Linux | amd64 |[tidb-latest-linux-amd64.sha256](http://download.pingcap.org/tidb-latest-linux-amd64.sha256)|
-| [tidb-binlog-kafka-linux-amd64.tar.gz](http://download.pingcap.org/tidb-binlog-kafka-linux-amd64.tar.gz)（Kafka 版本的 TiDB-Binlog）| Linux | amd64 |[tidb-binlog-kafka-linux-amd64.sha256](http://download.pingcap.org/tidb-binlog-kafka-linux-amd64.sha256)|
+| [tidb-latest-linux-amd64.tar.gz](http://download.pingcap.org/tidb-latest-linux-amd64.tar.gz) (TiDB Binlog) | Linux | amd64 |[tidb-latest-linux-amd64.sha256](http://download.pingcap.org/tidb-latest-linux-amd64.sha256)|
+| [tidb-binlog-kafka-linux-amd64.tar.gz](http://download.pingcap.org/tidb-binlog-kafka-linux-amd64.tar.gz)（Kafka 版本的 TiDB Binlog）| Linux | amd64 |[tidb-binlog-kafka-linux-amd64.sha256](http://download.pingcap.org/tidb-binlog-kafka-linux-amd64.sha256)|
+
+## TiDB Lightning
+
+如需下载最新版本的 [TiDB Lightning](/reference/tools/tidb-lightning/overview.md), 点击下表中的下载链接即可。
```
Comment (lilin90) [inline v3.0/reference/tools/download.md]:
```suggestion
如需下载最新版本的 [TiDB Lightning](/reference/tools/tidb-lightning/overview.md)，点击下表中的下载链接即可。
```

File: v3.0/reference/tools/download.md [hunk 3]
```diff
@@ -8,16 +8,24 @@ aliases: ['/docs-cn/tools/download/']
 
 本页面汇总了 TiDB 周边工具官方维护版本的下载链接。
 
-## TiDB-Binlog 和 Lightning
+## TiDB-Binlog
```
Comment (lilin90) [inline v3.0/reference/tools/download.md]:
```suggestion
## TiDB Binlog
```

File: v2.1/reference/tools/download.md [hunk 4]
```diff
@@ -7,16 +7,24 @@ category: reference
...
+
+| 安装包 | 操作系统 | 架构 | SHA256 校验和 |
+|:---|:---|:---|:---|
+| [tidb-toolkit-latest-linux-amd64.tar.gz](http://download.pingcap.org/tidb-toolkit-latest-linux-amd64.tar.gz) | Linux | amd64 | [tidb-toolkit-latest-linux-amd64.sha256](http://download.pingcap.org/tidb-toolkit-latest-linux-amd64.sha256) |
 
 ## TiDB-DM (Data Migration)
```
Comment (lilin90) [inline v2.1/reference/tools/download.md]:
```suggestion
## TiDB DM (Data Migration)
```

File: v3.0/reference/tools/download.md [hunk 5]
```diff
@@ -8,16 +8,24 @@ aliases: ['/docs-cn/tools/download/']
...
+
+## TiDB Lightning
+
+如需下载最新版本的 [TiDB Lightning](/reference/tools/tidb-lightning/overview.md), 点击下表中的下载链接即可。
+
+| 安装包 | 操作系统 | 架构 | SHA256 校验和 |
+|:---|:---|:---|:---|
+| [tidb-toolkit-latest-linux-amd64.tar.gz](http://download.pingcap.org/tidb-toolkit-latest-linux-amd64.tar.gz) | Linux | amd64 | [tidb-toolkit-latest-linux-amd64.sha256](http://download.pingcap.org/tidb-toolkit-latest-linux-amd64.sha256) |
 
 ## TiDB-DM (Data Migration)
```
Comment (lilin90) [inline v3.0/reference/tools/download.md]:
```suggestion
## TiDB DM (Data Migration)
```

File: v2.1/reference/tools/download.md [hunk 6]
```diff
@@ -7,18 +7,26 @@ category: reference
...
+| [tidb-binlog-kafka-linux-amd64.tar.gz](http://download.pingcap.org/tidb-binlog-kafka-linux-amd64.tar.gz)（Kafka 版本的 TiDB Binlog）| Linux | amd64 |[tidb-binlog-kafka-linux-amd64.sha256](http://download.pingcap.org/tidb-binlog-kafka-linux-amd64.sha256)|
 
-## TiDB-DM (Data Migration)
+## TiDB Lightning
+
+如需下载最新版本的 [TiDB Lightning](/reference/tools/tidb-lightning/overview.md)，点击下表中的下载链接即可。
+
+| 安装包 | 操作系统 | 架构 | SHA256 校验和 |
+|:---|:---|:---|:---|
+| [tidb-toolkit-latest-linux-amd64.tar.gz](http://download.pingcap.org/tidb-toolkit-latest-linux-amd64.tar.gz) | Linux | amd64 | [tidb-toolkit-latest-linux-amd64.sha256](http://download.pingcap.org/tidb-toolkit-latest-linux-amd64.sha256) |
```
Comment (kennytm) [inline v2.1/reference/tools/download.md]: (Same comment as https://github.com/pingcap/docs/pull/1337/#pullrequestreview-261723369)

Comment (ericsyh) [reply v2.1/reference/tools/download.md]: In v2.1 doc, i changed the tag from latest to v2.1.0 and do not separate the TiDB Lightning from the TiDB pkg. commit: https://github.com/pingcap/docs-cn/pull/1628/commits/0b96b756224ae5a5a714ae992cdfa41d9e7b96a2
In v3.0 doc, i changed the tag from latest to v3.0.0. commit: https://github.com/pingcap/docs-cn/pull/1628/commits/48534185fb86329a0150bb372ea79691b3ee72a6

**Pull request merged**